domain/src/main/java/com/thoughtworks/go/config/materials/git/GitMaterial.java

```diff
@@ -50,20 +51,18 @@
 import static org.apache.commons.lang3.StringUtils.isBlank;
 
 public class GitMaterial extends ScmMaterial implements PasswordAwareMaterial {
-    private static final Logger LOG = LoggerFactory.getLogger(GitMaterial.class);
     public static final int UNSHALLOW_TRYOUT_STEP = 100;
     public static final int DEFAULT_SHALLOW_CLONE_DEPTH = 2;
     public static final String GO_MATERIAL_BRANCH = "GO_MATERIAL_BRANCH";
-
-    private UrlArgument url;
-    private String branch = GitMaterialConfig.DEFAULT_BRANCH;
-    private boolean shallowClone = false;
-    private String submoduleFolder;
-
     //TODO: use iBatis to set the type for us, and we can get rid of this field.
     public static final String TYPE = "GitMaterial";
-    private static final String ERR_GIT_NOT_FOUND = "Failed to find 'git' on your PATH. Please ensure 'git' is executable by the Go Server and on the Go Agents where this material will be used.";
     public static final String ERR_GIT_OLD_VERSION = "Please install Git-core 1.6 or above. ";
+    private static final Logger LOG = LoggerFactory.getLogger(GitMaterial.class);
+    private static final String ERR_GIT_NOT_FOUND = "Failed to find 'git' on your PATH. Please ensure 'git' is executable by the Go Server and on the Go Agents where this material will be used.";
+    private final UrlArgument url;
+    private String refSpecOrBranch = GitMaterialConfig.DEFAULT_BRANCH;
```

Comment:
Renamed only the field so it's clear what it really points to and how it should be interpreted/treated. Did not rename the getter for fear of changing serialization or something.